backend/utils/threads_utils.py

```diff
@@ -0,0 +1,43 @@
+from http.client import HTTPException
+import json
+from typing import Any, Optional
+from schema.message import Thread
+from schema.message import Message, MessageRequest
+
+from utils.db import DataStorage
+from utils.message_utils import get_message ,update_message
+from config.settings import settings
+
+# List  all messages in a thread
+
+async def get_messages_in_thread(org_id, room_id, message_id):
+
+    # fetch message parent of the thread
+    DB = DataStorage(org_id)
+    messages = await get_message(org_id, room_id, message_id)
+    threads = messages["threads"]
+
+    return threads
```

Comment:
```suggestion
    return messages["threads"]
```
You can then remove the line above